kv/db.go

```diff
@@ -146,3 +147,78 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", contentType)
 	w.Write(body)
 }
+
+// RegisterRPC ...
+func (s *DBServer) RegisterRPC(rpcServer *rpc.Server) error {
+	return rpcServer.RegisterName("Server", &s.rpcDBServer)
```

Comment:
Yeah, it is confusing. Note that the "Node." endpoints differ from the "Server." endpoints due to the different senders: `Node` uses a `LocalSender` while `DBServer` uses `TxnCoordSender` wrapped around a `DistSender`.